it/uk/gov/hmrc/soletraderidentificationfrontend/controllers/CheckYourAnswersControllerISpec.scala

```diff
@@ -108,50 +111,127 @@ class CheckYourAnswersControllerISpec extends ComponentSpecHelper
       }
     }
   }
+
   "POST /check-your-answers-business" should {
-    "redirect to continue url from the supplied journey config" in {
-
-      await(insertJourneyConfig(
-        journeyId = testJourneyId,
-        continueUrl = testContinueUrl,
-        optServiceName = None,
-        deskProServiceId = testDeskProServiceId,
-        signOutUrl = testSignOutUrl,
-        enableSautrCheck = testEnableSautrCheck
-      ))
-      stubRetrieveSoleTraderDetails(testJourneyId)(status = OK, body = testSoleTraderDetailsJson)
-      stubAuth(OK, successfulAuthResponse())
-      stubMatch(testSoleTraderDetails)(OK, successfulMatchJson(testSoleTraderDetails))
-
-
-      lazy val result = post(s"/identify-your-sole-trader-business/$testJourneyId/check-your-answers-business")()
-
-      result must have(
-        httpStatus(SEE_OTHER),
-        redirectUri(s"$testContinueUrl?journeyId=$testJourneyId")
-      )
+    "redirect to continue url from the supplied journey config" when {
+      "the user has only provided a nino" in {
```

Comment:
Could we include that this is the non SAUTR journey, not where a user has skipped entering an SAUTR please?